git-cliff-core/src/gitlab.rs

```diff
@@ -0,0 +1,315 @@
+use crate::config::Remote;
+use crate::error::*;
+use futures::{
+	future,
+	stream,
+	StreamExt,
+};
+use http_cache_reqwest::{
+	CACacheManager,
+	Cache,
+	CacheMode,
+	HttpCache,
+	HttpCacheOptions,
+};
+use reqwest::header::{
+	HeaderMap,
+	HeaderValue,
+};
+use reqwest::Client;
+use reqwest_middleware::{
+	ClientBuilder,
+	ClientWithMiddleware,
+};
+use secrecy::ExposeSecret;
+use serde::de::DeserializeOwned;
+use serde::{
+	Deserialize,
+	Serialize,
+};
+use std::env;
+use std::hash::{
+	Hash,
+	Hasher,
+};
+use std::time::Duration;
+
+/// GitLab REST API url.
+const GITLAB_API_URL: &str = "https://gitlab.com/api/v4/";
+
+/// Environment variable for overriding the GitLab REST API url.
+const GITHUB_API_URL_ENV: &str = "GITLAB_API_URL";
```

Comment:
I recon this should be `GITLAB_API_URL_ENV`?

Reply:
Sure!